src/app/blog/page.tsx

```diff
@@ -49,22 +54,46 @@ const blogPageStructuredData: WithContext<WebPage> = {
   },
 }
 
+function getMetaDataContent(post: BlogPostData) {
```

Comment:
I feel like passing any JSX to the `metadata` prop of `PageHeader` gives too much flexibility to the consumer of the component.

I would type `metadata` as `Array<string | null | undefined>` and include the JSX snippet directly in the component, such as:

```tsx
// PageHeader.tsx
{metaData && metaData.length > 0 && (
  <div className="mb-6 flex gap-3 font-bold text-brand-300">
    {metaData.filter(Boolean).map((data, index) => {
      return (
        <p key={index}>
          {data}
          {index < metaData.length - 1 && (
            <span className="px-2">|</span>
          )}
        </p>
      )
    })}
  </div>
```

```tsx
// Blog Page
<PageHeader
  metaData={[formatDate(post.publishedOn), categoryLabel]}
/>
```

This would make sure the styling stays consistent across pages. Thoughts?

Reply:
Love this. Will implement it.

Reply:
@CharlyMartin - Let me know what you think [2341b41](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/49/commits/2341b4108b2f0089fd1513da1b340aceb596a212)